docs/source/index.rst

```diff
@@ -0,0 +1,362 @@
+
+    .. code-block:: python
+
+        {
+        # tags specific to building1
+        '/campus1/building1':
+            {
+            'site': true,
+            'dis': ": 'some building description',
+            'yearBuilt': 2015,
+            'area': '24000sqft'
+            },
+        # tags that apply to all device of a specific type
+        '/campus1/building1/deviceA*':
+            {
+            'dis': "building1 chilled water system - CHW",
+            'equip': true,
+            'campusRef':'campus1',
+            'siteRef': 'campus1/building1',
+            'chilled': true,
+            'water' : true,
+            'secondaryLoop': true
+            }
+        # tags that apply to point1 of all device of a specific type
+        '/campus1/building1/deviceA*/point1':
+            {
+            'dis': "building1 chilled water system - point1",
+            'point': true,
+            'kind': 'Bool',
+            'campusRef':'campus1',
+            'siteRef': 'campus1/building1'
+            }
+        # tags that apply to point2 of all device of a specific type
+        '/campus1/building1/deviceA*/point2':
+            {
+            'dis': "building1 chilled water system - point2",
+            'point': true,
+            'kind': 'Number',
+            'campusRef':'campus1',
+            'siteRef': 'campus1/building1'
+            }
+        # tags that apply to point3 of all device of a specific type
+        '/campus1/building1/deviceA*/point3':
+            {
+            'dis': "building1 chilled water system - point3",
+            'point': true,
+            'kind': 'Number',
+            'campusRef':'campus1',
+            'siteRef': 'campus1/building1'
+            }
+        # tags that apply to all device of a specific type
+        '/campus1/building1/deviceB*':
+            {
+            'dis': "building1 device of type B",
+            'equip': true,
+            'chilled': true,
+            'water' : true,
+            'secondaryLoop': true,
+            'campusRef':'campus1',
+            'siteRef': 'campus1/building1'
+            }
+        # tags that apply to point1 of all device of a specific type
+        '/campus1/building1/deviceB*/point1':
+            {
+            'dis': "building1 device B - point1",
+            'point': true,
+            'kind': 'Bool',
+            'campusRef':'campus1',
+            'siteRef': 'campus1/building1',
+            'command':true
+            }
+        # tags that apply to point1 of all device of a specific type
+        '/campus1/building1/deviceB*/point2':
+            {
+            'dis': "building1 device B - point2",
+            'point': true,
+            'kind': 'Number',
+            'campusRef':'campus1',
+            'siteRef': 'campus1/building1'
+            }
+        }
+
+Step 2: Create tags using template above
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Make an RPC call to the add_tags method and pass the python dictionary object
```

Comment:
probably should show an example of this.

Reply:
added

Reply:
probably should create a variable and pass it rather than inline dictionary....just do a couple of them.

Reply:
done